GliaCoreDependency/1.0/GliaCoreDependency.podspec

```diff
@@ -0,0 +1,15 @@
+Pod::Spec.new do |s|
+  s.name         = 'GliaCoreDependency'
+  s.version      = '1.0'
+  s.summary      = 'Glia Core SDK dependency'
+  s.description  = 'The Glia Core SDK brings the in-person customer experience to iOS devices.'
+  s.homepage     = 'https://github.com/salemove/ios-bundle'
+  s.license      = { :type => "MIT", :file => "LICENSE.md" }
+  s.author       = "ReactiveCocoa"
```

Comment:
Is this correct name for `author`?

Reply:
good catch. fixed - d84996a8473654a2fadacb1134cd898bf0294297